website/src/plugins/github-service.ts

```diff
@@ -140,4 +140,67 @@ export class GitHubService {
       throw err;
     }
   }
+
+  /**
+   * Fetches the latest contributors from the repository.
+   * @param excludeUsernames - Set of usernames to exclude (case-insensitive; usernames will be normalized internally)
+   * @param contributorLimit - Maximum number of contributors to return
+   * @param pagesToFetch - Number of pages of commits to fetch (default: 5)
+   * @returns Array of contributors sorted by commit count
+   */
+  public async getLatestContributors(
+    excludeUsernames: Set<string>,
+    contributorLimit: number,
+    pagesToFetch = 5,
+  ): Promise<Contributor[]> {
+    // Normalize exclude set to lowercase for case-insensitive comparison
+    const normalizedExcludeUsernames = new Set([...excludeUsernames].map(username => username.toLowerCase()));
+    // GitHub API limits to 100 per page, so we fetch multiple pages
+    const allCommits: Awaited<ReturnType<typeof this.octokit.rest.repos.listCommits>>['data'] = [];
+
+    for (let page = 1; page <= pagesToFetch; page++) {
+      const { data: commits } = await this.octokit.rest.repos.listCommits({
+        owner: this.owner,
+        repo: this.repo,
+        per_page: 100,
+        page,
+      });
+      allCommits.push(...commits);
+
+      // Stop if we got fewer than 100 commits (no more pages)
+      if (commits.length < 100) {
+        break;
+      }
+    }
+
+    // Group commits by author and filter out maintainers and bots
+    const contributorMap = new Map<string, Contributor>();
+
+    // Helper function to check if a username is a bot
+    const isBot = (login: string): boolean => login.endsWith('[bot]');
```

Comment:
issue: it means podman-desktop-bot would be kept ?, maybe needs a endsWith('-bot') as well